src/metaschema/examples/cis-sp-800-53-mapping.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- This is a mapping example used for development. This file should be moved to the oscal-content repo when this feature is ready. -->
+<mapping-collection xmlns="http://csrc.nist.gov/ns/oscal/1.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://csrc.nist.gov/ns/oscal/1.0 oscal-complete.xsd" uuid="3559d200-4849-41ac-a420-28b2ffa22c52">
+    <metadata>
+        <title>Example mapping between CIS controls and SP 800-53 rev5</title>
+        <last-modified>2022-04-13T08:37:21.323321800-04:00</last-modified>
+        <version>0.0.1</version>
+        <oscal-version>1.0.3</oscal-version>
+    </metadata>
+    <mapping uuid="9eb2019c-f3be-4f96-947e-58876a46b2a9">
+        <source-resource type="catalog" href="#a84961de-55ae-4bf3-a2d3-86cc32b651af"></source-resource>
+        <target-resource type="catalog" href="#711085f6-c390-4b25-b5f1-30066a56073d"></target-resource>
+        <map uuid="6a9a1161-770e-4556-9740-41e1809e14ea">
+            <relationship>equal-to</relationship>
+            <source type="control" id-ref="#cis-1.1"/>
+            <target type="control" id-ref="#cm-8">
+                <!-- TODO: consider a way to reference parameters allowing the review period of at least bi-annually to be described -->
+                <!-- <using-param id="cm-08_odp.02">at least bi-annually</using-param>-->
+            </target>
+            <target type="control" id-ref="#cm-8.1"/>
+            <remarks>
+                <p>The combination of SP 800-53 CM-8 and CM-8(1) describe similar implementation requirements to CIS 1.1.</p>
```

Comment:
The relation is `equal-to` but the remark lists a "similar implementation requirement". equal is not "similar". It can be described as 'equivalent' requirement.

Reply:
Indeed. Discussed this with CIS and we also agreed that `equivalent-to` is better. I'll make the suggested change above and update the example(s) and schema when I make edits next.